sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/serialization/ModelProperties.java

```diff
@@ -0,0 +1,64 @@
+package com.azure.digitaltwins.core.serialization;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Properties on a component that adhere to a specific model.
+ */
+public class ModelProperties {
```

Comment:
`@Fluent` here, as well

Reply:
Does this add any functionality? I haven't used this annotation before